### PR TITLE
initial code and readme for blog post

### PR DESCRIPTION
**Pull request by codeforger-wp**
For brevity, I am unconcerned with Sanitization of inputs, or escaping of outputs. Unless one of you is particularly keen on it.

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<?php
+require_once 'object-singleton.php';
+require_once 'static-singleton.php';
+
+sleep( 1 );
+?>
+
```
Comment (iamthomasbarrett) [inline index.php]: Best not to have whitespace before doctype.

File: index.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+				background-color: #eee;
+			}
+		</style>
+	</head>
+	<body>
+
+		<h1> Requester Info </h1>
+		<p>This page demonstrates the calling conventions for object singletons and static classes.</p>
+		<p>We expect that both columns have the same values, as we are demonstrating that these two techniques for global data containers are equivalent.</p>
+		<table>
```
Comment (iamthomasbarrett) [inline index.php:27]: Did you want to make full use of thead and tbody too?

File: object-singleton.php [hunk 3]
```diff
@@ -0,0 +1,60 @@
+<?php
+
```
Comment (iamthomasbarrett) [inline object-singleton.php:2]: Do you want it to be commented?

Comment (codeforger-wp) [reply object-singleton.php:2]: As the code is explained in full in the blog post, do you think its also worth having comments in the code? I guess if people land directly on this code base, then comments might be usefull. Ill add some

**Review by jamesmorrisonpragmatic — APPROVED** [review]

**Pull request merged**